backend/database/images.py

```diff
@@ -102,7 +102,19 @@ def thumbnail(self):
 
             pil_image = self.generate_thumbnail()
             pil_image = pil_image.convert("RGB")
-            pil_image.save(thumbnail_path)
+
+            # Calculate resize ratio:
+            resize_ratio = min(1, self.MAX_THUMBNAIL_DIM[0] / pil_image.height,
+                               self.MAX_THUMBNAIL_DIM[1] / pil_image.width)
```

Comment:
I thing we can simplify this by using `PIL.Image.thumbnail`

Reply:
Yeah, you're right. I can make the changes on Monday and update the pull request.